cljfmt/src/cljfmt/main.clj

```diff
@@ -136,8 +136,12 @@
              (warn "Failed to format file:" (project-path options f))
              (print-stack-trace e))))))))
 
-(def ^:private cli-file-reader
-  (comp edn/read-string slurp diff/to-absolute-path))
+(defn- cli-file-reader [opt-value]
+  (let [filename (diff/to-absolute-path opt-value)
```

Comment:
What's the purpose of converting the filename to an absolute path?

Reply:
Well, I just blindly copied `cli-file-reader`'s source code. I made a slight [change](https://github.com/weavejester/cljfmt/pull/202/commits/ffc2dcc11eec599a5b6abb255b5e57330722c17f#diff-a57e762da42583224b62656b3ae05addR144-R145), so `filename-ext` receives the raw string, but let me know if you meant something else.

Reply:
Ah, I see. I'm not sure why the `diff/to-absolute-path` is there to begin with. I think we can safely remove it.

Reply:
Great! Yes, I can't see a reason for it either. I just [removed it](https://github.com/weavejester/cljfmt/pull/202/commits/334ef64029fb132249b89f7f06af3d8aa0d76a8f).